### DISCUSSION TITLE
Request for Quire Website Feedback

### DISCUSSION THREAD

**Discussion by Erin-Cecele** [thegetty/quire#121] [Feedback]
As of May 2021, the new Quire website is live at **https://quire.getty.edu**. 

We will continue to update the site, both in content and appearance, over the next few months as we prepare to launch Quire as a fully open-source publishing tool by spring 2022. In the meantime, we'd love your feedback so we can make this site the best possible resource for the community. 

We invite you to share your thoughts, impressions, and any other feedback below. 

Thank you!

**Comment by antoinentl** [comment]
First of all, kudos to the team for this amazing work!

2 questions:

-  perhaps add a source/modify link in every page of the doc, to encourage users to contribute to updates of the doc? I can purpose a pull request if you think it's relevant;
- a journal/blog section (something more written than the news section) will be very interesting, to track changes (new feature, human version of a changelog, new example in the showcase, etc.) and follow the evolution of the project (I'm going to do that for several projects in [the lab](https://ecrituresnumeriques.ca/en/) where I work) with a classic RSS/Atom feed.

**Reply by Erin-Cecele** [reply]
Thank you, @antoinentl. That is very helpful feedback! 

It would be great if you could create a pull request for your idea of adding a source/modify link on the documentation pages. To get started, please review our [CONTRIBUTING.md](https://github.com/thegetty/quire-docs/blob/main/CONTRIBUTING.md) file. We ask that you first propose this feature as a [new issue](https://github.com/thegetty/quire-docs/issues/new) on our https://github.com/thegetty/quire-docs repository's [issue tracker](https://github.com/thegetty/quire-docs/issues). This lets others know that you are working on it and gives the rest of the Quire team a heads up. We look forward to seeing your work.

And a journal/blog section on the website is a very interesting idea. Thank you for suggesting that.

**Reply by antoinentl** [reply]
Ok I follow your recommandations, I create the issue and the merge request, feel free to comment when ypu/the team have the time, no rush.
